relationships.md

```diff
@@ -39,8 +39,8 @@ The leftId and rightId parameter in JSON may still be present, but they will be
   "_embedded": {
     "relationshipType": {
       "id": 1,
-      "leftLabel": "isAuthorOfPublication",
-      "rightLabel": "isPublicationOfAuthor",
+      "leftWardLabel": "isAuthorOfPublication",
+      "rightWardLabel": "isPublicationOfAuthor",
```

Comment:
Small thing, but I don't believe the "W" needs to be capitalized?  The terms are "leftward" and "rightward".  So, I think we can rename these to `leftwardLabel` and `rightwardLabel` (throughout this PR).

Reply:
That's correct, I've updated all usages